service/templates/task.html

```diff
@@ -344,10 +344,16 @@ <h3 class="section-heading"><i class="fa fa-exchange"></i> Depends On</h3>
           </div>
           <div class="col-lg-5">
             <div class="row">
-              <div class="col-md-4">
+              <div class="col-md-3">
                 <h3 class="section-heading"><i class="fa fa-flask"></i> Tests</h3>
               </div>
-              <div class="col-md-8">
+              <div class="col-md-5">
+                <div ng-hide="!task.test_results || task.test_results.length == 0"
+                class="pull-right" style="margin-top:15px">
+                      <input type="text" class="form-control" ng-change="filterTests()" ng-model="task.searchField" placeholder="Filter tests"/>
```

Comment:
I worry that iterating through all the results on each keystroke will perform poorly for large sets of data. Could you experiment a bit with adding 100000 dummy test results in the UI and see how the search responds to that?

Reply:
I tested this with 1000 results, then with 10000. In both cases, there was not a noticeable change in performance.